app/services/get_application_forms_waiting_for_provider_decision.rb

```diff
@@ -0,0 +1,12 @@
+class GetApplicationFormsWaitingForProviderDecision
+  def self.call
+    ApplicationChoice
+      .where(status: :awaiting_provider_decision)
+      .joins(:application_form)
+      .where(['application_forms.submitted_at < ?', 20.business_days.ago])
```

Comment:
The constant 20 is effectively part of the business rules, because it's a counterpart to the 40-day limit. I think the 20 should be defined in [`TimeLimitConfig`](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/lib/time_limit_config.rb) and pulled into here.